src/extensions/geo-export/usdz-exporter.ts

```diff
@@ -32,17 +31,21 @@ export class UsdzExporter extends MeshExporter<UsdzData> {
     readonly fileExtension = 'usdz';
     private meshes: string[] = [];
     private materials: string[] = [];
-    private materialSet = new Set<number>();
+    private materialMap = new Map<string, number>();
     private centerTransform: Mat4;
 
-    private static getMaterialKey(color: Color, alpha: number) {
-        return color * 256 + Math.round(alpha * 255);
-    }
+    private addMaterial(color: Color, alpha: number, metalness: number, roughness: number): number {
```

Comment:
Make this an object? (it's at the random 4 args limit, but at the same time it's all numbers which could be easily mistaken)

Reply:
Maybe, it's just a private method that is used once. Is there a performance penalty of using objects?

Reply:
Ok, fine as it is.